src/textures/VideoFrameTexture.js

```diff
@@ -6,6 +6,8 @@ class VideoFrameTexture extends VideoTexture {
 
 		super( {}, mapping, wrapS, wrapT, magFilter, minFilter, format, type, anisotropy );
 
+		this.isVideoFrameTexture = true;
```

Comment:
Please document the new property.

Reply:
done

Reply:
Not like, sorry. We don't use JSDoc in these classes yet. Please add the doc here:

https://rawcdn.githack.com/mrdoob/three.js/dev/docs/index.html#api/en/textures/VideoFrameTexture